animations/effects.json

```diff
@@ -324,5 +324,22 @@
 				}
 			]
 		}
+	],
+	"effect:spell-effect-shields-of-the-spirit-security": [
+		{
+			"trigger": "effect",
+			"preset": "onToken",
+			"file": "jb2a.markers.shield_rampart.complete.03.blue",
+			"options": {
+				"scaleToObject": {
+					"value": 1.5
+				},
+				"persist": {
```

Comment:
When making persistent animations, make sure to create a lesser version of the effect for lower-end machines. See Marshal's Aura for implementation examples.

Reply:
Oh, I only saw it on marshal so wasn't sure if this applied